### PR TITLE
Update pl_pl.json

### PR DESCRIPTION
**Pull request by SpiderKolo**
I looked into Polish translation and I through I would bring in few tweaks and changes.
Some of the translations were a bit too direct and didn't feel to fluid, so I replaced few of them to be less direct while still expressing their accurate meaning.

There is also a slight issue (tho it's also present in original, English language) where it's difficult to distinguish between Origin and Minecraft Mob (Phantom and... yeah you get the idea). I decided to tweak a bit those to make them slightly more distinct.

Majority of the translation was left untouched, but there were few missing translations which I filled holes for.

It would be nice if original translator took a look for secondary opinion. He did a good job overall and I trust his opinion.

### CODE REVIEW

File: src/main/resources/assets/origins/lang/pl_pl.json [hunk 1]
```diff
@@ -143,17 +143,17 @@
   "power.origins.shulker_inventory.name": "Zbieracz",
   "power.origins.shulker_inventory.description": "Masz dostęp do dodatkowych 9 miejsc w ekwipunku, które zatrzymują przedmioty po śmierci.",
 
-  "power.origins.natural_armor.name": "Solidna Skóra",
-  "power.origins.natural_armor.description": "Nawet bez zbroi twoja skóra zapewnia naturalną ochronę.",
+  "power.origins.natural_armor.name": "Pancerz",
+  "power.origins.natural_armor.description": "Twoja skóra jest jak pancerz zapewniający ci naturalną ochronę, nawet gdy nie nosić faktycznej zbroi.",
```
Comment (K0-RR) [inline src/main/resources/assets/origins/lang/pl_pl.json:147]: misspelling - should be `nosisz`

File: src/main/resources/assets/origins/lang/pl_pl.json [hunk 2]
```diff
@@ -143,17 +143,17 @@
...
   "power.origins.more_exhaustion.name": "Obżartuch",
   "power.origins.more_exhaustion.description": "Wyczerpujesz się znacznie szybciej niż inni, przez co musisz jeść więcej.",
 
   "power.origins.no_shield.name": "Niezręczny",
-  "power.origins.no_shield.description": "Sposób, w jaki uformowane są twoje ręce, nie pozwala na trzymanie tarczy w pozycji pionowej.",
+  "power.origins.no_shield.description": "Sposób, w jaki zbudowane są twoje dłonie, nie pozwala na odpowiednie trzymanie tarczy. Nie możesz jej unieść by osłonić się przed atakami.",
 
   "power.origins.strong_arms.name": "Silne Ramiona",
-  "power.origins.strong_arms.description": "Jesteś wystarczająco silny, aby wykopywać kamień naturalny* bez użycia kilofa. * - nie postawiony przez graczy.",
+  "power.origins.strong_arms.description": "Jesteś wystarczająco silny aby wydobywać naturalnie występujący kamień bez użycia kilofa.",
```
Comment (K0-RR) [inline src/main/resources/assets/origins/lang/pl_pl.json:156]: Ohh great you changed these. My version was awful.

**Comment by K0-RR** [comment]
Original translator here, I approve all these changes, many of them are better versions of my translations.

**Comment by apace100** [comment]
This has been merged manually into the 1.17 branch. :)

**Pull request closed without merging**